internal/postprocessor/execv/gocmd/gocmd.go

```diff
@@ -105,18 +105,21 @@ func Build(dir string) error {
 	return nil
 }
 
-// Vet runs linters on all .go files recursively from the given directory.
-func Vet(dir string) error {
-	log.Println("vetting generated code")
+// Format runs goimports on all .go files recursively from the given directory.
+func Format(dir string) error {
+	log.Println("formatting generated code")
 	c := execv.Command("goimports", "-w", ".")
 	c.Dir = dir
 	if err := c.Run(); err != nil {
 		return err
 	}
-
-	c = execv.Command("gofmt", "-s", "-d", "-w", "-l", ".")
```

Comment:
Can you add a bullet point in the commit explaining why gofmt is no longer desired, just for future reference?

Reply:
added a note to the commit, good idea!